src/coreclr/dlls/mscordac/DacToWindbgLongName/DacToWindbgLongName.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
```

Comment:
Should these be `NetCoreAppToolCurrent` so that these tools continue to work during major version bumps?